go.mod

```diff
@@ -1,20 +1,23 @@
 module github.com/asecurityteam/component-connstate
 
-go 1.19
+go 1.22
+
+toolchain go1.23.1
 
 require (
-	github.com/asecurityteam/component-stat v0.5.0
-	github.com/asecurityteam/settings v0.7.0
-	github.com/golang/mock v1.5.0
-	github.com/stretchr/testify v1.8.2
+	github.com/asecurityteam/component-stat v0.5.2
+	github.com/asecurityteam/settings v1.0.0
```

Comment:
Do we need settings v1 if we're including v2?

Reply:
omg good catch! I need to make that manual change!